### PR TITLE
Add bowei to OWNERS: e2e/test dns,network; cloud route, node, service…

### PR DESCRIPTION
**Pull request by bowei**

### CODE REVIEW

**Comment by bowei** [comment]
@mikedanese @janetkuo : approval?

File: pkg/controller/OWNERS [hunk 1]
```diff
@@ -4,5 +4,6 @@ approvers:
 - derekwaynecarr
 - mikedanese
 - janetkuo
+- bowei # for node,route, service controllers
```
Comment (mikedanese) [inline pkg/controller/OWNERS]: why don't you add yourself to OWNERS files in these packages?

Comment (bowei) [inline pkg/controller/OWNERS]: done

**Pull request merged**